### ISSUE TITLE
Question about layernorm?

### ISSUE THREAD

**Issue by senseandcalm** [131250208/JointExtraction#37]
line 86 in components.py:
`std = (variance + self.epsilon) **2`
Is this a bug? I think it should be:
`std = (variance + self.epsilon) ** 0.5`

**Comment by 131250208** [comment]
Yes, you are right. But it might not influence too much. You can fix it by yourself.
